Signal/src/ViewControllers/ConversationView/Cells/OWSSystemMessageCell.m

```diff
@@ -25,18 +25,12 @@ @interface OWSSystemMessageCell ()
 @property (nonatomic) UIImageView *imageView;
 @property (nonatomic) UILabel *titleLabel;
 
-// override from JSQMessagesCollectionViewCell
-@property (nonatomic) UILabel *cellTopLabel;
```

Comment:
Obsolete view.

Reply:
This happened a long time ago but I think we agreed to this design change.  I think they should be able to use message details if they really wanted to see when this happened.